apps/examples/nextjs/package.json

```diff
@@ -7,9 +7,9 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "12.1.0",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
+    "next": "12.3.4",
```

Comment:
While we are at it can we bump this to the latest `13.x` version?

Reply:
Make sure to run `pnpm add --filter nextjs-example react@latest react-dom@latest next@latest` in folder root :)

Reply:
It's fine if the lock-file changes again. Don't mind the `settings` block being added, that's expected as you mentioned

Reply:
I see, thanks. About the `settings` entry in the lockfile. It was introduced in [8.6.0](https://github.com/pnpm/pnpm/releases/tag/v8.6.0), but it was rolled back in (https://github.com/pnpm/pnpm/releases/tag/v8.6.2)[8.6.2], so nevermind my comment